scss/civicrm/_variables.scss

```diff
@@ -27,6 +27,9 @@ $crm-light-gray-color: #818c97;
 
 $font-family-fontawesome: 'FontAwesome';
 
+$crm-font-weight-normal: 400;
+$crm-font-weight-bold: 600;
```

Comment:
Bold weight is equivalent of 700.
Check https://htmldog.com/references/css/properties/font-weight/#fbol

Reply:
This is just a name of variable, we have "bold" font set as 600. Or would you have any suggestions for the variable name?